src/jit/compiler.h

```diff
@@ -8843,6 +8843,8 @@ void * CompAllocator::ArrayAlloc(size_t elems, size_t elemSize)
 inline
 LclVarDsc::LclVarDsc(Compiler* comp)
     :
+#ifndef _TARGET_X86_
+    // TODO-x86 - Need initialization of the _lvArgReg and _lvOtherArgReg for x86?
```

Comment:
Well... It looks like when used as x86 Jit, the liveness analysis fails the assert above. It looks like a bug in the liveness analysis. When RyuJit is ported to x86 the issue needs/will be addressed. Removing the initialization stops the seertion failures.